src/instance.ts

```diff
@@ -29,6 +29,11 @@ import {
   GetAppProfilesCallback,
   GetAppProfilesResponse,
 } from './app-profile';
+import {
```

Comment:
how come this is not on `table.ts`? In other clients, all backup operations are on the table admin client. ex: https://github.com/liubonan/java-bigtable/blob/master/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java#L805. Let me know if my understanding is incorrect :)

Reply:
Hello! We ended up with these methods defined throughout various classes:

#### Access or create Backup objects through a Cluster reference
```js
cluster.backup(id) // references a pre-existing "Backup" object (no remote calls)
cluster.createBackup({ id: 'backup-id', table: 'table-id' }) // creates and returns a "Backup" object
cluster.getBackups() // returns "Backup" objects
cluster.getBackupsStream() // returns "Backup" objects as a readable stream
```

#### Access Backup objects through an Instance reference
```js
instance.createTableFromBackup({ table: 'table-id', backup: 'backup-id' }, gaxOptions)
instance.getBackups() // returns "Backup" objects
instance.getBackupsStream() // returns "Backup" objects as a readable stream
```

#### Create a Backup through a Table reference
```js
table.createBackup(id, metadata) // creates and returns a "Backup" object
```

#### Once you have a reference to a Backup object:
*All methods below besides "restore()" are common CRUD methods defined throughout our API*
```js
backup.create({ table: 'table-id', metadata: {...} })
backup.delete()
backup.get()
backup.getMetadata()
backup.restore('table-id')
backup.setMetadata({...})
```